client/network/sync/src/lib.rs

```diff
@@ -463,7 +463,7 @@ where
 				// If there are more than `MAJOR_SYNC_BLOCKS` in the import queue then we have
 				// enough to do in the import queue that it's not worth kicking off
 				// an ancestor search, which is what we do in the next match case below.
-				if self.queue_blocks.len() > MAJOR_SYNC_BLOCKS.into() {
+				if self.queue_blocks.len() > MAJOR_SYNC_BLOCKS as usize {
```

Comment:
Why is there such a change, this seems very unrelated to your stuff?

Reply:
here i get a `type annotations needed` error with

multiple `impl`s satisfying `usize: PartialOrd<_>` found in the following crates: `core`, `substrate_fixed`

i dont fully understand the exact issue here, somehow the compiler is confused because `substrate_fixed` brings some new impls. what would be the proper way to address this @clangenb ?

Reply:
Ah, ok. This is something that we can't resolve then. Then I would just add a comment (in the PR not the code) about why this change was introduced to prevent any questions.